### ISSUE TITLE
no longer works, just attempts to load reddit and flashes loading.

### ISSUE THREAD

**Issue by AtmaDarkwolf** [xPuls3/New-Reddit-Redirect#3]
Only able to load the 'new' ugly reddit. 

Dec 11 2024 (Was working before reddit updated 'privacy' issues. IDK if direct cause or not)

**Comment by Aetherinox** [comment]
Yeah, there was an announcement on Reddit that they are completely killing new.reddit.com today, it's gone for good.

**Comment by m-doescode** [comment]
Sucks... I wonder if an extension could be written that retrieves the old-new reddit's js files from archive.org and loads those instead of the garbage that is new-new reddit's
